static-checks/ansible/allowed-modules/test.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python3
+
+import os
+import re
+import subprocess
+import tempfile
+import yaml
+import contextlib
+from pathlib import Path
+
+from lib import util, results
+
+
+# Obtained from
+# https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#task
+ansible_reserved_keywords = ['action', 'any_errors_fatal', 'args', 'async', 'become', 'become_exe',
+                             'become_flags', 'become_method', 'become_user', 'changed_when',
+                             'check_mode', 'collections', 'connection', 'debugger', 'delay',
+                             'delegate_facts', 'delegate_to', 'diff', 'environment', 'failed_when',
+                             'ignore_errors', 'ignore_unreachable', 'local_action', 'loop',
+                             'loop_control', 'module_defaults', 'name', 'no_log', 'notify', 'poll',
+                             'port', 'register', 'remote_user', 'retries', 'run_once', 'tags',
+                             'throttle', 'timeout', 'until', 'vars', 'when', 'with_items']
```

Comment:
Would it make sense to reformat this into something more manageable?
```python
ansible_reserved_keywords = [
    'action',
    'any_errors_fatal',
    'args',
    'async',
    ...
]
```
or at least avoid the indent, and use breaking at 99-char lines?
```python
ansible_reserved_keywords = [
    'action', 'any_errors_fatal', 'args', 'async', 'become', 'become_exe', 'become_flags',
    'become_method', 'become_user', 'changed_when', 'check_mode', 'collections', 'connection',
    ...
]
```
?